### PR TITLE
CAT Instructors Updates [WIP]

### PR DESCRIPTION
**Pull request by nomadscientist**

### CODE REVIEW

**Comment by nomadscientist** [comment]
- [ ] meetnjoin not working
- [ ] awspolly - should we have a Lupe?
- [ ] Is stuff like 'video lecture' translatable?
- [ ] Intro-to-galaxy automated slide video (I made the painstaking line-by-line translation, needs automated video linked)
- [ ] Ditto for the intro-to-scRNA-seq lecture video

**Comment by nomadscientist** [comment]
@shiltemann This is looking ok actually!

**Comment by nomadscientist** [comment]
Human translations (Looking at links only)

- [ ] For scRNA-intro-lecture - the video works (https://training.galaxyproject.org/training-material/videos/watch.html?v=transcriptomics/tutorials/scrna-intro/slides_ES) but somehow isn't working on the page
- [ ]

**Comment by shiltemann** [comment]
> CAT - auto-translation
> 
>     * [ ]  meetnjoin not working
I will have a look at that
 
>     * [ ]  awspolly - should we have a Lupe?
AWS Polly is the name of the service, not one of the individual voices, though I do like the idea of listing the voice used (but maybe less of a priority)

>     * [ ]  Is stuff like 'video lecture' translatable?
yes, this should just be taken from the "type" field of the video yaml
 
>     * [ ]  Intro-to-galaxy automated slide video (I made the painstaking line-by-line translation, needs automated video linked)
>     * [ ]  Ditto for the intro-to-scRNA-seq lecture video
@hexylena could you upload the video to youtube? (or did you already?)


> General issues
> 
>     * [ ]  Create some sort of reference link so we can have 1 list of instructors/trainers and then refer to it in each of the programmes and course pages, rather than retyping it 5 times and having to keep it updated?
If it's just all of the instructors for every session, you can leave out the instructors altogether, this was mainly for smorgasbord where we had 100 instructors and wanted to give them extra credit. If you are okay with just listing everybody at bottom of the page you can delete the `instructors:` key and nothing will be shown and that might be ok? If not, then yes, it is possible to define it once and include it multiple places, I can put in an example of that

>     * [ ]   Line 58 - buttons to switch languages!
>       
>       * Can we get two large buttons at the bottom of each page to say ENG-English, 'CAT-Español' and 'HES-Español' [whichever 2 are needed]? And then link somehow so we can track how often those particular buttons are used?
Link to switch to one of the other programs you mean right? We can setup analytics with plausible, track how many times each page was opened, will have to check if you can also see which buttons were used to navigate there, unsure, might need google analytics for that?

>     * [ ]  Wherever it says 'chat with us @gxytraining we should add 'And tag us a #spanscriptomics on twitter' or whatever
will add support for this!

>     * [ ]  We need an anchor in our course page (index.md) at the 'Porque' section that describes the study. I want to link to that from the first text bit in each programme. Same again for the index-en.md (Why?)
>     * [x]  Me - HREC approval and English doc 
>     * [ ]  HREC project information translated and then uploaded
>     * [ ]  weird commas in instructors etc.
should be fixed by unifying instructor names with the contributors.yaml file

>     * [x]  Me - SUPPORT LINKS
>     * [x]  Stefy - how to update your preferences at the top
>     * [ ]  Logos are not rendering
>     * [ ]  Turn the 'post-course survey' (link pending) into a large button?
can do! 
>     * [ ]  W/Stefy - link to post-course survey
> 
> 
> Human translations - HES
> 
>     * [ ]  For scRNA-intro-lecture - the video works (https://training.galaxyproject.org/training-material/videos/watch.html?v=transcriptomics/tutorials/scrna-intro/slides_ES) but somehow isn't working on the page
we should upload that to youtube if not already done, and link to that.
>     * [x]  All other tutorial & video links work!
>     * [ ]  Rendering VERY weird though, missing lots of info. Help?
>     * [ ]  Translations

**Comment by nomadscientist** [comment]
`> Link to switch to one of the other programs you mean right? We can setup analytics with plausible, track how many times each page was opened, will have to check if you can also see which buttons were used to navigate there, unsure, might need google analytics for that?`
Yes - that's perfect with plausible. Don't worry about which buttons, we also are surveying that question, so that's fine. Plausible and buttons please :)

**Comment by hexylena** [comment]
> Link to switch to one of the other programs you mean right? We can setup analytics with plausible, track how many times each page was opened, will have to check if you can also see which buttons were used to navigate there, unsure, might need google analytics for that?

yeah, just keep the utm_campaign to 'spanscriptomics' and vary the utm_medium or so

**Comment by nomadscientist** [comment]
@hexylena Cool - I made the links and popped them in, but making them big buttons is something I cannot (yet) do!

File: events/spanscriptomics/program-cat.md [hunk 1]
```diff
@@ -88,8 +88,8 @@ schedule:
         title: "Quieres cambiar idioma?"
         description: |
           Aquí puede cambiar a los programas con recursos que fueron traducidos automáticamente (ENG-Inglés) o traducidos por humanos (HES-Español). ¡Recuerde anotar esto en la encuesta posterior al curso! ¡Estos son datos valiosos!
-          [ENG-Inglés](https://gallantries.github.io/galaxy-workshop/events/spanscriptomics/program-eng/)
-          [HES-Español](https://gallantries.github.io/galaxy-workshop/events/spanscriptomics/program-hes/)
+          [ENG-Inglés](https://gallantries.github.io/galaxy-workshop/events/spanscriptomics/program-eng?utm_campaign=Spanscriptomics&utm_medium=pagebutton)
```
Comment (hexylena) [inline events/spanscriptomics/program-cat.md:91]:
```suggestion
          [ENG-Inglés](https://gallantries.github.io/galaxy-workshop/events/spanscriptomics/program-eng?utm_campaign=Spanscriptomics&utm_medium=program-CAT&utm_source=pagebutton)
```

do something like this for each. CAT here, HES in program-hes

Comment (nomadscientist) [reply events/spanscriptomics/program-cat.md:91]: ooooooooo - on it!

**Comment by hexylena** [comment]
> HES
> * Rendering VERY weird though, missing lots of info. Help?

could you explain here a bit?

**Comment by nomadscientist** [comment]
> > HES
> > 
> > * Rendering VERY weird though, missing lots of info. Help?
> 
> could you explain here a bit?

Yes -I'm hoping it's just my Github Pages but basically it only renders this:
Screenshot 2021-11-22 at 12 11 31

**Comment by hexylena** [comment]
Where did you get that URL?

https://gallantries.github.io/galaxy-workshop/events/spanscriptomics/program-es is the currently deployed version, I'm not sure about what it looks like locally currently

**Comment by hexylena** [comment]
Discovered an issue with the automated videos today, the second slide wasn't being translated. Only affects scRNA currently but I'll need to regenerate those after we figure out what went wrong

image

**Comment by nomadscientist** [comment]
> Discovered an issue with the automated videos today, the second slide wasn't being translated. Only affects scRNA currently but I'll need to regenerate those after we figure out what went wrong
> 
> image

Which videos was this for? the HES or the CAT?

**Comment by shiltemann** [comment]
> Which videos was this for? the HES or the CAT?

It will probably be for both, but this is one of the slides that are part of the template, not something you can control. We will have to look at that on the framework level.

**Review by shiltemann — APPROVED** [review]

**Pull request merged**